<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>TID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Active Waiting</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Events</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Status">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Suspend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Idle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lowest</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Below_Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Above_Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Highest</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Critical</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Priority</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
